### DISCUSSION TITLE
HOW TO ENABLE STUDENT DEVELOPER PACK

### DISCUSSION THREAD

**Discussion by pullurugeethika** [community/community#39572] [GitHub Education]
I dont know please anyone help me

**Comment by pullurugeethika** [comment]
its showing request is in pending

**Comment by SilasRodrigues19** [comment]
@pullurugeethika Just wait for them to review the request.

**Comment by DSDmark** [comment]
Hi @pullurugeethika,

in my knowledge,

> As a student, you may be requested to submit proof of enrollment at an educational institution. Verification can take up to a few weeks - GitHub will provide an estimated timeframe after you apply

Here read this:- (https://github.com/orgs/community/discussions/22701)

**Reply by egendaryl** [reply]
加油

**Reply by DSDmark** [reply]
?? 
come on hmm....
